addons/mail/static/src/messaging/component/partner_im_status_icon/partner_im_status_icon.xml

```diff
@@ -8,11 +8,14 @@
                 'o-bot': partner and env.messaging.partnerRoot === partner,
                 'o-offline': partner and partner.im_status === 'offline',
                 'o-online': partner and partner.im_status === 'online',
-            }" t-att-data-partner-local-id="partner ? partner.localId : undefined"
-        >
+                'o_PartnerMentionSuggestionDisplay': props.partnerMentionSuggestionDisplay,
+            }"
+            t-att-data-partner-local-id="partner ? partner.localId : undefined" >
             <t t-if="partner" name="rootCondition">
-                <i class="o_PartnerImStatusIcon_outerBackground fa fa-circle fa-stack-1x"/>
-                <i class="o_PartnerImStatusIcon_innerBackground fa fa-circle fa-stack-1x"/>
+                <t t-if="!props.partnerMentionSuggestionDisplay">
```

Comment:
Partner im status icon should not have any concern for mention suggestion at all.
This props should act as a customization for this component. Here, it should either enable or disable background, so the name of props should have something with that, e.g. `hasBackground`